README.md

```diff
@@ -198,22 +263,38 @@ Then deploy the `dist` folder
 
 ## Integration & Validation Tests
 
-For the integration tests to work, you have to launch the fake API and the dashboard first. Run the following commands in separate terminal sessions:
+For the integration tests to work, you have to first run the localstack application, launch the localAPI and serve the dashboard first. Run the following commands in separate terminal sessions:
 
+Run background localstack application.
 ```bash
-  $ node fake-api.js
-  $ APIROOT=http://localhost:5001 npm run serve
+  $ npm run start-localstack
 ```
 
-Run the test suite in another terminal:
+Serve the cumulus API (separate terminal)
+```bash
+  $ npm run serve-api
+```
 
+Serve the dashboard web application (another terminal)
 ```bash
-  $ npm run validation
+  $ npm run serve
```

Comment:
jk here is where you should change the command to include the correct APIROOT

Reply:
8a0bde66